systemd/openqa-minion-restart.service

```
@@ -0,0 +1,6 @@
+[Unit]
+Description=Restarts services which are using Minion
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/systemctl try-restart openqa-webui.service openqa-gru.service openqa-worker-cacheservice-minion.service
```

We probably also need to restart `openqa-worker-cachservice.service` as it is basically the Minion dashboard on the workers:
```suggestion
ExecStart=/usr/bin/systemctl try-restart openqa-webui.service openqa-gru.service openqa-worker-cacheservice.service openqa-worker-cacheservice-minion.service
```

---

Added.